integration/tests/schema_helpers_test.go

```diff
@@ -0,0 +1,57 @@
+// +build integration
+
+package tests
+
+import (
+	"context"
+	"fmt"
+
+	openapi "github.com/asecurityteam/asset-inventory-api/client"
+)
+
+var schemaVersion int32  //current schema version
+var maxSchema int32 = 14 // TODO: extrapolate this somewhere?
```

Comment:
nit: rename to maxSchemaVersion

Reply:
This is copy/paste from what was master. Git for some reason does not reflects it. I'll change the name :)

Reply:
Done, also switched to const.